src/lib/common/string.cpp

```diff
@@ -95,7 +95,6 @@ bool isIPv6(const std::string& in)
 {
   size_t      pos;
   std::string partip;
-  std::string resu;
```

Comment:
I understand that this is a case of variable not used, detected by cppcheck.

The code is supposed to be compiled with gcc flags to fail with error when variable not used is detected. I wonder why the compiler has not complaint about this...

Anyway, NTC, good catches.